self-hosted/Dockerfile

```diff
@@ -63,18 +71,17 @@ RUN set -x \
   && mkdir -p $SENTRY_CONF
 
 COPY . .
-RUN python3 -m tools.fast_editable --path .
-RUN sentry help | sed '1,/Commands:/d' | awk '{print $1}' >  /sentry-commands.txt
+RUN : \
+    && python3 -m tools.fast_editable --path . \
+    && sentry help | sed '1,/Commands:/d' | awk '{print $1}' >  /sentry-commands.txt
 
 COPY ./self-hosted/sentry.conf.py ./self-hosted/config.yml $SENTRY_CONF/
 COPY ./self-hosted/docker-entrypoint.sh /
 
 RUN : double-check some built files are available \
     && test -f /usr/src/sentry/src/sentry/loader/_registry.json \
     && test -f /usr/src/sentry/src/sentry/integration-docs/python.json \
-    && test -f /usr/src/sentry/src/sentry/static/sentry/dist/entrypoints/app.js \
-    && sentry help \
```

Comment:
this is still needed -- we don't have pipefail in the earlier command

please avoid changing unrelated things